capsules/src/usb/cdc.rs

```diff
@@ -63,6 +63,34 @@ enum State {
     Connected,
 }
 
+/// States of the Control Endpoint related to CDC-ACM.
+#[derive(Debug, Copy, Clone, PartialEq)]
+enum CtrlState {
+    /// No ongoing ctrl transcation.
+    Idle,
+    /// Host has sent a SET_LINE_CODING configuration request.
+    SetLineCoding,
+}
+
+#[derive(PartialEq)]
+enum CDCCntrlMessage {
```

Comment:
Nit pick: "ctrl" is a much more common abbreviation for "control". Otherwise, is saving two letters worth it over fully spelling "control"?